client_with_ssl.js

```diff
@@ -1,52 +1,38 @@
 'use strict';
 
-const {Client} = require('hazelcast-client');
-const fs = require('fs');
-const path = require('path')
-
-function createClientConfig() {
-    return {
-        network: {
-            hazelcastCloud: {
-                discoveryToken: 'YOUR_CLUSTER_DISCOVERY_TOKEN'
-            },
-            ssl: {
-                enabled: true,
-                sslOptions: {
-                    ca: [fs.readFileSync(path.resolve(path.join(__dirname, 'ca.pem')))],
-                    cert: [fs.readFileSync(path.resolve(path.join(__dirname, 'cert.pem')))],
-                    key: [fs.readFileSync(path.resolve(path.join(__dirname, 'key.pem')))],
-                    passphrase: 'YOUR_SSL_PASSWORD',
-                    rejectUnauthorized: false
-                }
-            }
-        },
-        clusterName: 'YOUR_CLUSTER_NAME',
-        properties: {
-            'hazelcast.client.cloud.url': 'YOUR_DISCOVERY_URL',
-            'hazelcast.client.statistics.enabled': true,
-            'hazelcast.client.statistics.period.seconds': 1,
-        }
-    }
+const {Client, HazelcastJsonValue} = require('hazelcast-client');
+
+async function mapExample(client){
+    const cities = await client.getMap('cities');
+    await cities.put(1, new HazelcastJsonValue(JSON.stringify({ country: "United Kingdom", city: "London", population: 9_540_576})));
+    await cities.put(2, new HazelcastJsonValue(JSON.stringify({ country: "United Kingdom", city: "Manchester", population: 2_770_434})));
+    await cities.put(3, new HazelcastJsonValue(JSON.stringify({ country: "United States", city: "New York", population: 19_223_191})));
+    await cities.put(4, new HazelcastJsonValue(JSON.stringify({ country: "United States", city: "Los Angeles", population: 3_985_520})));
+    await cities.put(5, new HazelcastJsonValue(JSON.stringify({ country: "Turkey", city: "Ankara", population: 5_309_690})));
+    await cities.put(6, new HazelcastJsonValue(JSON.stringify({ country: "Turkey", city: "Istanbul", population: 15_636_243})));
+    await cities.put(7, new HazelcastJsonValue(JSON.stringify({ country: "Brazil", city: "Sao Paulo", population: 22_429_800})));
+    await cities.put(8, new HazelcastJsonValue(JSON.stringify({ country: "Brazil", city: "Rio de Janeiro", population: 13_635_274})));
+
+    const mapSize = await cities.size();
+    console.log(`'cities' map now contains ${mapSize} entries.`);
+
+    console.log("--------------------");
 }
+async function sqlExample(hzClient) {
```

Comment:
Add the comment add a newline please 
```suggestion

async function sqlExample(hzClient) {
```